value_lists/vl_document_type.sql

```diff
@@ -0,0 +1,23 @@
+/*
+	qWat - QGIS Water Module
+
+	SQL file :: document_type table
+*/
+
+/* CREATE */
+CREATE TABLE qwat_vl.document_type () INHERITS ( qwat_vl.value_list_base);
+ALTER TABLE qwat_vl.document_type ADD CONSTRAINT vl_document_type_pk PRIMARY KEY (id);
+COMMENT ON TABLE qwat_vl.document_type IS 'document_type table';
+
+/* COLUMNS */
+ALTER TABLE qwat_vl.document_type ADD COLUMN active     boolean default true; /* determines if the actual document_type is active or not */
+
+INSERT INTO qwat_vl.document_type (id, value_en, value_fr, value_ro ) VALUES (101, 'other', 'autre', 'alta');
+INSERT INTO qwat_vl.document_type (id, value_en, value_fr, value_ro ) VALUES (102, 'unknown', 'inconnu', 'necunoscută');
+INSERT INTO qwat_vl.document_type (id, value_en, value_fr, value_ro ) VALUES (103, 'to be determined', 'à déterminer', 'de determinat');
+
+/* VALUES */
+INSERT INTO qwat_vl.document_type (id, active, value_en, value_fr) VALUES (8001, true, 'Picture', 'Picture');
+INSERT INTO qwat_vl.document_type (id, active, value_en, value_fr) VALUES (8002, true, 'PDF', 'PDF');
```

Comment:
I would not say that PDF is a document type.

If appliable I would better have it set to a doc type list like qgep :

https://user-images.githubusercontent.com/8614223/70708118-0b85fc80-1cda-11ea-966b-0c8456d310c9.png

If it is thought for a functionnality, I would rather use file extensions and not enter some manual attribute.

Reply:
I completed the list according to the QGEP list (without voice recording for now).